safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -200,28 +225,34 @@ def handle(self, *args, **options):
             self.stdout.write(
                 self.style.SUCCESS(f"Setting up {ethereum_network.name} safe addresses")
             )
-            self._setup_safe_master_copies(MASTER_COPIES[ethereum_network])
+            self._setup_safe_singleton_addresses(MASTER_COPIES[ethereum_network])
+            self._setup_erc20_indexing()
+        else:
```

Comment:
`self.stdout.write` should be called warning people that indexing will be started from scratch (block 0) as we are autodetecting the deployments